src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/AstNodes/XamlNotSharedDeferredContentNode.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System.Linq;
+using XamlX.Ast;
+using XamlX.Emit;
+using XamlX.IL;
+using XamlX.Transform;
+using XamlX.TypeSystem;
+
+namespace Avalonia.Markup.Xaml.XamlIl.CompilerExtensions.AstNodes;
+
+class XamlNotSharedDeferredContentNode : XamlAstNode, IXamlAstValueNode, IXamlAstEmitableNode<IXamlILEmitter, XamlILNodeEmitResult>
```

Comment:
This is an almost exact copy of `DeferredContentNode`, except the `_deferredContentCustomization field` being initialized in the constructor. Please delete this and reuse `DeferredContentNode`.